vllm/model_executor/layers/linear.py

```diff
@@ -59,7 +65,7 @@ def __init__(self, separate_bias_add: bool = False):
     def create_weights(self, input_size_per_partition: int,
                        output_size_per_partition: int, input_size: int,
                        output_size: int,
-                       params_dtype: torch.dtype) -> Dict[str, Any]:
+                       params_dtype: torch.dtype, logical_widths: Optional[List[int]]) -> Dict[str, Any]:
```

Comment:
lift this to be inside `LinearMethodBase` ?

Reply:
I got rid of this on the next pr